.eslintrc.json

```diff
@@ -0,0 +1,20 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true
+    },
+    "extends": [
+        "google"
+    ],
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
```

Comment:
I think this is referencing new es features that I doubt we use: https://tc39.es/ecmascript_sharedmem/shmem.html#GlobalObject.ConstructorProps

Reply:
Fixed.